joomla/joomla-cms
# [com_fields] Properly sanitise "alias" in field

Pull Request for Issue #14497 .

### Summary of Changes
Passing the item language to the stringURLSafe method in the field table


### Testing Instructions
Save a field and check that the alias is sanitised as expected. In english there should be no change at all to the current behavior. In other languages transliteration may kick in depending on the field language (or site language) setting.
